schemas/lerna-schema.json

```diff
@@ -0,0 +1,1833 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://lerna.js.org/docs/getting-started#adding-lerna",
+  "title": "JSON schema for Lerna configuration",
+  "type": "object",
+  "properties": {
+    "version": {
+      "type": "string",
+      "description": "The version of the repository, or \"independent\" for an repository with independently versioned packages.",
+      "default": "0.0.0"
+    },
+    "packages": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "uniqueItems": true
+      },
+      "description": "An array of globs where packages can be located.",
+      "default": ["packages/*"]
+    },
+    "useNx": {
+      "type": "boolean",
+      "description": "When true, Lerna will use the nx task runner.",
```

Comment:
```suggestion
      "description": "When true, Lerna will delegate the implementation details of `lerna run` to the Nx task runner, instead of p-map or p-queue.",
```

Reply:
Updated 👍